demos/textfield.html

```diff
@@ -194,12 +210,73 @@
           }, 0);
         </script>
       </section>
+
+      <section class="example">
+        <h2>Textfield - Leading/Trailing icons</h2>
+        <div class="demo-note">
+          <em>
+            Note: Some implementations utilizing leading and trailing icons may wish to set a width on the
+            <code>input</code> element to achieve uniform widths on textfields.
+            This demo achieves that by adding a custom class name: <code>custom-textfield-input</code> and setting
+            some style:
+          </em>
+          <br />
+          <!--
+            The following text formatting achieves the visual output we are aiming for
+            since the <pre> tag honors whitespace and line breaks.
+          -->
+          <pre>
+<code>.custom-textfield-input {
+  width: 300px;
+}</code></pre>
+        </div>
+        <div id="demo-tf-box-leading-wrapper">
+          <div id="tf-box-leading-example"
+            class="mdc-textfield mdc-textfield--box mdc-textfield--with-leading-icon" data-demo-no-auto-js>
+            <i class="material-icons mdc-textfield__icon" tabindex="0">event</i>
+            <input type="text" id="tf-box-leading" class="mdc-textfield__input custom-textfield-input">
+            <label for="tf-box-leading" class="mdc-textfield__label">Your Name</label>
+            <div class="mdc-textfield__bottom-line"></div>
```

Comment:
Is it intentional that bottom-line is after label in the textfield box example, but before label in the non-box example?

Reply:
Not intentional, I'll make them uniform